examples/production/values.yaml

```diff
@@ -2,6 +2,7 @@ global:
   telemetry:
     prometheus:
       enabled: true
+  jwtIssuer: https://oidc-discovery.example.org
```

Comment:
Maybe this in charts/spire/values.yaml instead?

Reply:
I figured this would create less pushback and ensure my charts aren't broken in future upgrades :D

Reply:
Its already being set to oidc-discovery.example.org in charts/spire/values. if its wrong to not have the https://, then it should probably be corrected there?